src/mumble/PositionalAudioViewer.cpp

```diff
@@ -0,0 +1,74 @@
+// Copyright 2021 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#include "PositionalAudioViewer.h"
+#include "ui_PositionalAudioViewer.h"
+
+#include "PluginManager.h"
+#include "Global.h"
+
+PositionalAudioViewer::PositionalAudioViewer() : m_ui(new Ui::PositionalAudioViewer) {
```

Comment:
`std::make_unique` is useful on assignment, in this case we're calling the constructor.

Reply:
It's about semantics and less about usefulness. If I look at this code in the current state I immediately look into the destructor and become worried because there is no matching `delete` statement. On the other hand if I look at the `make_unique` version I immediately see that `m_ui` is a smart-pointer that cleans up after itself automatically